### ISSUE TITLE
Proposal to Create a privacy policy for this project 

### ISSUE THREAD

**Issue by frankremmy** [FaisalUmair/udemy-downloader-gui#70]
Hello, I found your project quite amazing. I want to contribute by providing a privacy policy for this project. The question is will you review and merge it to your repository when it is done? If so, tell me so I can get started as soon as possible. Thanks

**Comment by frankremmy** [comment]
Yes. I'll have to communicate with you in other to create a quality privacy policy. Also guidance and what should be added or not. My discord username is Ufrankremmy#2995

**Comment by frankremmy** [comment]
Or any other platform for communication

**Comment by FaisalUmair** [comment]
Hi @frankremmy. You can go ahead and create one on your own. Once it is ready I will review it. 

And regarding what should be added. I think the most important part is to mention that the only input the application takes from a user is their username and password and the same is sent to Udemy server for authentication. Any other data that can be accessed after authentication is evident in the app. The only information accessed after authentication is the courses related to the user account. No other data regarding the user's Udemy account is gathered.

**Comment by frankremmy** [comment]
Got it! It seems you've been really busy these days. I've gotten what I need. I'll just have to get to work, once I'm done, I'll pull request, you review it and then merge if you see it fit.
